routers/web/repo/actions/actions.go

```diff
@@ -424,3 +426,30 @@ func decodeNode(node yaml.Node, out any) bool {
 	}
 	return true
 }
+
+func DeleteRun(ctx *context.Context) {
```

Comment:
Unless I'm mistaken, we also need to remove the `ActionTask`, `ActionTaskStep`, and `ActionTaskOutput` c.f. https://github.com/go-gitea/gitea/pull/26275#issuecomment-1667072215 ?

Reply:
Yes, I think so, and it needs some tests.